### DISCUSSION TITLE
Discuss the new badge pattern

### DISCUSSION THREAD

**Discussion by yesenia-perezcruz** [Shopify/polaris-react#6751] [Pattern documentation]
## Original issue summary

There's confusion about when to use a New badge and what the New badge should look like. 

![Screen Shot 2022-06-10 at 3 53 14 PM (1)](https://user-images.githubusercontent.com/47306956/173142910-bac3bce9-2e09-4a93-ab47-6f25904f6837.png)

<img width="833" alt="image (2)" src="https://user-images.githubusercontent.com/47306956/173142985-57265420-db6a-4424-9a6f-08a17360302a.png">

Guidance we should include about when to use is:

"New" should be used on features that have all of these 3 characteristics:
1. We want to drive adoption to this feature;
2. Is creating net new value for merchants (not just an improved way of doing something they are already doing);
3. Is worth distracting our merchants away from their current workflow to inform them about a whole new part of Shopify.

**Comment by jared-fanning** [comment]
initial guidance published: [https://polaris.shopify.com/foundations/patterns/new-badge](https://polaris.shopify.com/foundations/patterns/new-badge)

**Comment by selenehinkley** [comment]
Welcome to the new badge pattern discussion! Best practices around using a new badge (and notifications more broadly) are still being explored.

### Here are some things you can do here:

💡 **Share ideas** on how to make this pattern, or its documentation, better.

❓ **Ask a specific question** about this pattern. 

---
Tips before posting:
- Remember Polaris and our discussions are public. If you work for Shopify and have an externally sensitive topic, please reach out in #polaris in Slack
- Please [report bugs here](https://github.com/Shopify/polaris/issues/new?assignees=&labels=%F0%9F%90%9BBug&template=ISSUE.md)
- [Contributions](https://polaris.shopify.com/contributing) are always welcome
- Learn more about [Polaris discussions](https://github.com/Shopify/polaris/discussions/6750)

💜👩‍🚀✌️ the Polaris team
